harness.poplar

```diff
@@ -1,5 +1,4 @@
 // +build none
```

Comment:
Why were these files renamed to poplar? What does the poplar extension do?

Reply:
I didn't want the linter to hit them before we migrage them to poplar, which is ticketed.